app.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "Portier Broker",
+  "description": "Portier Broker reference implementation, written in Rust. Learn more at https://portier.github.io/.",
+  "keywords": [
+    "Portier",
+    "Passwordless",
+    "Authentication"
+  ],
+  "website": "https://portier.github.io/",
+  "repository": "https://github.com/portier/portier-broker",
+  "env": {
+    "BROKER_PUBLIC_URL": {
+      "description": "The broker server's public-facing URL. This will be the Issuer for JSON Web Tokens and Relying Parties will use the same value for their Broker URL. To ensure consistency, trailing slashes should be avoided. It is suggested that you choose an app name (up above) and then use it to fill out this config var. Alternatively, if you enable the Dyno Metadata labs addon using the Heroku CLI then you can leave this value blank and the public-facing URL will be automatically set, thus allowing you to let Heroku choose the app's name.",
+      "value": "https://[app-name].herokuapp.com"
+    },
+    "BROKER_KEYTEXT": {
+      "description": "The text of an encryption key in PEM format. You can create an encryption key by running `openssl genrsa 4096 > private.pem` and then provide the key to this broker by copying the file's contents here.",
+      "value": ""
+    },
+    "BROKER_FROM_ADDRESS": {
+      "description": "The 'From' address used by Portier to send emails. Unlike the BROKER_PUBLIC_URL, this config value will not auto-populate even if the Dyno Metadata labs addon is enabled. It will be easiest if you choose an app name (up above) and then use it here instead of letting Heroku pick a name for the app.",
+      "value": "portier@[app-name].herokuapp.com"
+    },
+    "BROKER_FROM_NAME": {
+      "description": "The 'From' name on the emails sent by Portier.",
+      "value": "Portier"
+    },
+    "BROKER_ALLOWED_ORIGINS": {
+      "description": "The domains allowed to use this broker for authentication. You can specify a single domain or list multiple domains and separate them with commas. Note there are no trailing slashes.",
+      "value": "https://example.com,https://example.org"
+    },
```

Comment:
Hello @stephank, thank you for the comments! The use of `origin` definitely makes more sense than `domain`; I'll make the change.

I also wanted to treat the allowed origins config as optional but I ran into trouble. It appears Heroku always sets an environment variable for each config var, even if that config is marked optional. If the user doesn't provide a value, Heroku uses the default we specify in our `app.json` (in this case "https://example.com,https://example.org", which probably isn't particularly accurate for most users :-).

We can perhaps work around this by moving the example values up into the description and leaving the default value empty (i.e. `"values": ""`). When I did this I ran into another difficulty: If the env var exists but is set to an empty string (i.e. `BROKER_ALLOWED_ORIGINS=""`) it appears the broker rejects any relying party's request to authenticate since none of them are in the allowed list. My very novice Rust eyes are drawn to `src/config.rs:440`; assuming I'm in the right area, maybe an "is not empty" check could be added?

What do you think?